cli/ballerina-cli/src/main/resources/cli-help/ballerina-graph.help

```diff
@@ -0,0 +1,45 @@
+NAME
+       ballerina-graph - Print the dependency graph
+
+SYNOPSIS
+       bal graph [OPTIONS] [<package>|<source-file>]
+
+
+DESCRIPTION
+       Resolve the dependencies of the current package and print the final
```

Comment:
```suggestion
       Resolve the dependencies of the current package and prints the
```

Reply:
Shouldn't this be either "Resolve ... and print" or "Resolves and prints" ?